heft-plugins/heft-lint-plugin/src/SarifFormatter.ts

```diff
@@ -303,12 +303,12 @@ export function formatEslintResultsAsSARIF(
         physicalLocation.region = region;
       }
 
-      if (message.source) {
-        physicalLocation.region ??= {};
-        physicalLocation.region.snippet = {
-          text: message.source
-        };
-      }
+      // if (message.source) {
+      //   physicalLocation.region ??= {};
+      //   physicalLocation.region.snippet = {
+      //     text: message.source
```

Comment:
Is `message.source` no longer a field?

Reply:
It is not... I'm going to poke around but it may be that this is no longer possible